docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -0,0 +1,190 @@
+.. _metrics_guide:
+
+====================
+Metrics Guide
+====================
+
+In this guide we'll explain how to customize the metrics that deepchecks uses to validate and monitor your model.
+Controlling the metrics helps you shape the checks and suites according to the specifics of you use case.
+
+**Structure:**
+
+* `Default metrics <#default-metrics>`__
+* `Alternative metrics <#alternative-metrics>`__
+* `List of Supported Strings <#list-of-supported-strings>`__
+* `Custom Metrics <#custom-metrics>`__
+
+
+Default metrics
+===============
+All of the checks that evaluate model performance, such as SingleDatasetPerformance, come with
+default metrics.
+
+The default scorers by task type are:
+
+Tabular
+_______
+
+Classification, binary or multiclass averaged over the classes:
```

Comment:
I think we should write here for each metric what is it's "deepchecks name", or what is the string that will produce it. For example, the recall in this section is "macro_averaged_recall" and the next one is "recall_per_class"